client/savePayment/html/src/recommended/index.html

```diff
@@ -17,7 +17,7 @@
     <h1>Save Payment Recommended Integration</h1>
 
     <div class="buttons-container">
-      <paypal-button id="paypal-button" type="pay" hidden></paypal-button>
+      <paypal-button id="paypal-button" hidden></paypal-button>
```

Comment:
I don't think the "Pay with" button label makes sense for saving a payment so removing it.